modules/data-streams/src/main/java/org/elasticsearch/datastreams/lifecycle/DataStreamLifecycleService.java

```diff
@@ -564,7 +814,7 @@ private void forceMergeIndex(ForceMergeRequest forceMergeRequest, ActionListener
         assert forceMergeRequest.indices() != null && forceMergeRequest.indices().length == 1
             : "Data stream lifecycle force merges one index at a time";
         final String targetIndex = forceMergeRequest.indices()[0];
-        logger.info("Data stream lifecycle is issuing a request to force merge index [{}]", targetIndex);
+        logger.trace("Data stream lifecycle is issuing a request to force merge index [{}]", targetIndex);
```

Comment:
Why change this one to `trace` level? We still log the downsampling message at `info` level?

Reply:
Ah, accidental I believe. Both forcemerge and downsampling log to info when executing inside the dedup.